src/odoc/bin/main.ml

```diff
@@ -742,8 +784,12 @@ module Depends = struct
     let info =
       Term.info "link-deps" ~docs:section_legacy
         ~doc:
-          "lists the packages which need to be in odoc's load path to link the \
-           .odoc files in the given directory"
+          "Lists the packages which need to be in odoc's load path to link the \
```

Comment:
'Forward references' isn't quite right - it's just as possible to miss a reference to a previously built package. Perhaps: "Lists a subset of the packages and modules which need to be in odoc's load path to link the $(i, odoc) files in the given directory. Additional packages may be required to resolve all references."